src/main/java/com/minecolonies/coremod/items/ItemSupplyChestDeployer.java

```diff
@@ -121,7 +121,7 @@ public EnumActionResult onItemUse(
             spawnShip(worldIn, pos, enumfacing);
             ItemStackUtils.changeSize(stack, -1);
 
-            playerIn.addStat(ModAchievements.achievementGetSupply);
+            //playerIn.addStat(ModAchievements.achievementGetSupply);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)